Gemfile

```diff
@@ -1,11 +1,12 @@
 source "https://rubygems.org"
 
-ruby "~> 3.2.0"
+ruby "~> 3.4.0"
 
 gem "rails", "8.0.2.1"
 
 gem "ast"
 gem "bootsnap", require: false
+gem "csv"
```

Comment:
Does this app use this gem?

Reply:
So it was in the lock file before, but actually doing a deeper dig, doesn't look like it is actually used anywhere!

I'll pull it out and see if it still works.